apiserver/environment.yml

```diff
@@ -0,0 +1,16 @@
+name: hail
```

Comment:
Is this the same requirements as hail/environment.yml? I'm concerned this is starting to be duplicated in multiple places and will be harder to maintain.

Reply:
It is not the same.  It includes dependencies for running Hail in the Jupyter notebook (normal Hail dependencies + Jupyter + requests for the ServiceBackend) and the dependencies for the apiserver (flask).

Conda images are incredibly huge so I wanted to share the same base between the notebook image and the apiserver.  As per our discussion in group meeting last week, I think dropping conda and installing Python and modules with pip directly will lead to much more manageable docker images.